index.bs

```diff
@@ -2224,11 +2224,11 @@ To <dfn>check if attribution debugging can be enabled</dfn> given an [=attributi
 To <dfn>serialize an attribution debug info</dfn> given a [=map=] |data| and an
 [=attribution debug info=] |debugInfo|:
 
-1. If |debugInfo|'s [=attribution debug info/source debug key=] is not null, [=map/set=]
-    |data|["`source_debug_key`"] to |debugInfo|'s [=attribution debug info/source debug key=],
+1. If |debugInfo|'s [=attribution debug info/source debug key=] is null, return.
```

Comment:
Should this just reuse the `check if attribution debugging can be enabled` algorithm?

Reply:
Updated, thanks.